exonum-java-binding/core/rust/src/storage/db.rs

```diff
@@ -91,6 +91,27 @@ impl View {
     pub fn get(&mut self) -> &mut ViewRef {
         &mut self.reference
     }
+
+    /// Unwraps the stored Fork from the View, panics if it's not possible.
+    pub fn into_fork(self) -> Box<Fork> {
+        if let Some(view_owned) = self._owned {
```

Comment:
I'm guessing if we should remove the underscore for the `_owned` field and update its description...